controllers/generateConfigController.js

```diff
@@ -33,7 +33,7 @@ const _promptBasePath = async () => {
   do {
     let hasError = false;
     basePath = promptUserInput(
-      'Base path for the translations files: e.g. "src/localizations.": '
+      'Base path for the translations files: e.g. "src/localizations.": ',
```

Comment:
Not sure if the point at `localizations.` is needed? Or add backstick maybe

Reply:
not needed hehe, you're right. thanks!